test/unit/lib/urlutilTestComponents.js

```diff
@@ -365,7 +365,7 @@ module.exports = {
       test.strictEqual(urlUtil().getOrigin('https://abc.bing.com'), 'https://abc.bing.com')
     },
     'gets URL origin for url with port': (test) => {
-      test.strictEqual(urlUtil().getOrigin('https://bing.com:443/?test=1#abc'), 'https://bing.com:443')
+      test.strictEqual(urlUtil().getOrigin('https://bing.com:8000/?test=1#abc'), 'https://bing.com:8000')
```

Comment:
this test fails in muon because muon gets the origin from GURL.GetOrigin (https://cs.chromium.org/chromium/src/url/gurl.h?q=gurl&sq=package:chromium&l=201), which doesn't return the port if it's 443 for HTTPS or 80 for HTTP